doc/release-notes.md

```diff
@@ -131,6 +131,10 @@ GUI changes
 - UTXOs which are locked via the GUI are now stored persistently in the
   wallet database, so are not lost on node shutdown or crash. (#23065)
 
+- Toggle chain from mainnet to testnet, signet or regtest. The GUI now always sets `-chain`,
+  so any occurrences of `-testnet`, `-signet` or `-regtest` in `bitcoin.conf` should
+  be replace with `chain=test`, `chain=signet` or `chain=regtest` respectively. (#414)
```

Comment:
Should I prefix the GitHub issue with `bitcoin-core/gui#414`?

Reply:
I guess so.

Reply:
Ok, done for for release notes. Not sure how to test the end result.